packages/libs/eda/src/lib/map/analysis/hooks/standaloneMapMarkers.tsx

```diff
@@ -154,15 +155,13 @@ export function useStandaloneMapMarkers(
     [boundsZoomLevel?.zoomLevel, geoConfig]
```

Comment:
I also noticed that I had done a missing dependency... tut tut.

I think it would be fixed by putting `boundZoomLevel?.zoomLevel != null` in place of `boundsZoomLevel` at line 146/147 above.

Reply:
Fixed it.